src/conn/conn_layered.c

```diff
@@ -1154,6 +1186,12 @@ __wti_disagg_conn_config(WT_SESSION_IMPL *session, const char **cfg, bool reconf
     if (__wt_conn_is_disagg(session)) {
         WT_ERR(__layered_table_manager_start(session));
 
+        /* If we are starting as a primary, abandon a previous incomplete checkpoint. */
+        if (leader) {
+            WT_WITH_CHECKPOINT_LOCK(session, ret = __disagg_abandon_checkpoint(session));
```

Comment:
This will abandon whichever checkpoint was last, correct?
How do we ensure it only drops an **incomplete** checkpoint and not a valid one?